prow/config/external_plugins_config.yaml

```diff
@@ -104,6 +104,7 @@ ti-community-label:
       - "needs-cherry-pick-release-7.1"
       - "needs-cherry-pick-release-7.2"
       - "needs-cherry-pick-release-7.3"
```

Comment:
Please delete these labels, same as branches, we just need keep the latest DMR's  need-cherry-pick label.

Reply:
Or make it updated in another PR.

Reply:
Will create a separate PR to handle the deletion of DMR-related content.